projects/valtimo/dossier/src/lib/dossier-assign-user/dossier-assign-user.component.ts

```diff
@@ -125,7 +125,7 @@ export class DossierAssignUserComponent implements OnInit, OnChanges, OnDestroy
             return a.lastName.localeCompare(b.lastName);
           }
         })
-        .map(user => ({text: `${user.firstName} ${user.lastName}`, id: user.id}))
+        .map(user => ({text: user.label, id: user.id}))
```

Comment:
I don't know how to feel about letting the API endpoint having a say in how a user is represented. @mritense do you have an opinion about this?